extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItemExtractor.java

```diff
@@ -58,4 +60,11 @@
      * Whether the uploader is verified by the service
      */
     boolean isUploaderVerified() throws ParsingException;
+
+    /**
+     * Get the reply extractor
+     */
+    CommentReplyExtractor getReplies() throws ParsingException;
+
+    boolean isAReply() throws ParsingException;
```

Comment:
Please rename this method and the corresponding variables.
```suggestion
    boolean isReply() throws ParsingException;
```